### ISSUE TITLE
Benchmark SKEIN

### ISSUE THREAD

**Issue by Jude-Masson** [Sniffdog/Sniffdogminer#161]
Hello
At me on algorithm the skein produces a hashrate of 3,59 GH / s. But the statistics prescribe 1.9 GH / s /. When you correct your hands with time, returns an understated value.

**Comment by Sniffdog** [comment]
oh you probably had a bad streak somewhere

goto Stats folder and delete Ccmineralexis_Skein_HashRate.txt file

Sniffdog will re-bench for you

**Comment by Jude-Masson** [comment]
Of course, I did, in the beginning.

Now the benchmark has been removed and launched.

> [2018-03-02 01:00:45] [S / A / T]: 0/42/42, diff: 15.100, 3476.66MH / s yes!
> [2018-03-02 01:00:47] [S / A / T]: 0/43/43, diff: 8.873, 3476.24MH / s yes!
> [2018-03-02 01:00:49] [S / A / T]: 0/44/44, diff: 10.916, 3476.31MH / s yes!

but what happened in the new file :)

>     "Live":  992400580,
>     "Minute":  992400580,
>     "Minute_Fluctuation":  0.5,
>     "Minute_5":  992400580,
>     "Minute_5_Fluctuation":  0.5,
>     "Minute_10":  992400580,
>     "Minute_10_Fluctuation":  0.5,
>     "Hour":  992400580,
>     "Hour_Fluctuation":  0.5,
>     "Day":  992400580,
>     "Day_Fluctuation":  0.5,
>     "Week":  992400580,
>     "Week_Fluctuation":  0.5,
>     "Updated":  "\/Date(1519941832697)\/"

So two algorithms behave.

On the same miner, the others work fine.

Sorry for my english using google translator

**Comment by Sniffdog** [comment]
wow twice?????

BADDog!!!! haha

that miner might be written in windows US english

**Comment by Jude-Masson** [comment]
And three times, I prescribed the values myself then put back.
Windows Russian, do you propose to install an English Windows?

**Comment by Sniffdog** [comment]
no don't install

**Comment by Jude-Masson** [comment]
I can change my hands back this once a day must be done.
I can not find in which file to change the StatsInterval 
I correctly understood that this is not only me, but also confirmed with you?

**Comment by Sniffdog** [comment]
https://github.com/Sniffdog/Sniffdogminer/archive/master.zip

then change this to -StatsInterval 10000 after....

![screenshot 44 _li](https://user-images.githubusercontent.com/30452757/36875108-d8064172-1d6b-11e8-9b08-848367d41100.jpg)

**Comment by Jude-Masson** [comment]
Thank you

**Comment by asupercoin** [comment]
Do i need to add -StatsInterval 10000 to my .bat file? I tested but the error occured.

C:\Users\accrypto\Downloads\MiningSoftware\Sniffdogminer-4.7-PhiPhi\Sniffdogminer-4.7\SniffDogBitcoin.ps1 : A parameter cannot be found that matches parameter name
'StatsInterval'.
At line:1 char:24
+ &.\SniffDogBitcoin.ps1 -StatsInterval 10000 -Workername AeeyChamp -Ri ...
+                        ~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [SniffDogBitcoin.ps1], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,SniffDogBitcoin.ps1

PS C:\Users\accrypto\Downloads\MiningSoftware\Sniffdogminer-4.7-PhiPhi\Sniffdogminer-4.7>

**Comment by Sniffdog** [comment]
yes phi phi is a fork of sniffdog and won't recognize the new command :)

**Comment by Sniffdog** [comment]
no it's in the master copy... not in releases yet
https://github.com/Sniffdog/Sniffdogminer/archive/master.zip
